src/macOSNotif.js

```diff
@@ -336,7 +310,7 @@ class macOSNotifJS {
 
         // Remove fully once animation completed
         setTimeout(() => {
-            this.container.parentElement.remove();
+            this.container.parentElement.removeChild(this.container);
```

Comment:
Should this not be `this.container.parentElement.parentElement.removeChild(this.container.parentElement);` to remove the parent on container (which is what is needed)?

Reply:
indeed, good catch